.github/workflows/build.yml

```diff
@@ -186,31 +188,31 @@ jobs:
         msan: [""]
         nBPF: [""]
         include:
-          - compiler: "gcc-7"
-            os: ubuntu-latest
```

Comment:
Is it really necessary to change `ubuntu-latest` to `ubuntu-20.04` and why (or why not) ?

Reply:
gcc-7 is no more available in (default ppa of) `ubuntu-22.04` (`ubuntu-latest` is "moving" target and it was `ubuntu-20.04` ans it is going to be `ubuntu-22.04`)
If we want to keep testing gcc-7|12 and clang-7|14 we need to force the OS version